javascript-data-model/order-history.js

```diff
@@ -0,0 +1,90 @@
+var orderHistory = [
+  {
+    orderNumber: 114 - 3941689 - 8772232,
+    orderDate: '8-4-2020',
+    ShippingAddress: 'JS Masher',
+    isDelivered: true,
+    DeliveryDate: '8-4-2020',
+    returnWindowOpen: false,
+    returnWindowCloseDate: '9-7-2020',
+    total: 34.00,
+    numItems: [{
+      category: 'book',
+      qty: 1,
+      itemName_or_title: 'JavaScript for Impatient Programmers',
+      author_or_manufacturer: 'Rauschmayer, Dr. Axel',
```

Comment:
Rather than having multiple possibilities in the name of this property, you should choose one or the other. The way this is written now, the value seems like it might be ambiguous.